### DISCUSSION TITLE
keri-dev 10/05/23

### DISCUSSION THREAD

**Discussion by m00sey** [WebOfTrust/keri#52] [Meetings]
##### issues (is:issue created:>2023-09-28)
- https://github.com/WebOfTrust/keripy/issues/582
- https://github.com/WebOfTrust/keripy/issues/583

##### prs (is:pr created:>2023-09-28)
- https://github.com/WebOfTrust/keripy/pull/580
- https://github.com/WebOfTrust/keripy/pull/581

##### issues (is:issue created:>2023-09-28)
None

##### prs (is:pr created:>2023-09-28)
- https://github.com/WebOfTrust/keria/pull/111
- https://github.com/WebOfTrust/keria/pull/112

##### issues (is:issue created:>2023-09-28)
None 

##### prs (is:pr created:>2023-09-28)
- https://github.com/WebOfTrust/signifypy/pull/82
- https://github.com/WebOfTrust/signifypy/pull/83

##### issues (is:issue created:>2023-09-28)
None

##### prs (is:pr created:>2023-09-28)
- https://github.com/WebOfTrust/signify-ts/pull/113
- https://github.com/WebOfTrust/signify-ts/pull/114
- https://github.com/WebOfTrust/signify-ts/pull/116

##### issues (is:issue created:>2023-09-28)
- https://github.com/hyperledger-labs/did-webs-resolver/issues/1
- https://github.com/hyperledger-labs/did-webs-resolver/issues/2
- https://github.com/hyperledger-labs/did-webs-resolver/issues/3
- https://github.com/hyperledger-labs/did-webs-resolver/issues/4
- https://github.com/hyperledger-labs/did-webs-resolver/issues/5
- https://github.com/hyperledger-labs/did-webs-resolver/issues/6
- https://github.com/hyperledger-labs/did-webs-resolver/issues/7
- https://github.com/hyperledger-labs/did-webs-resolver/issues/12
- https://github.com/hyperledger-labs/did-webs-resolver/issues/13
- https://github.com/hyperledger-labs/did-webs-resolver/issues/14
- https://github.com/hyperledger-labs/did-webs-resolver/issues/15
- https://github.com/hyperledger-labs/did-webs-resolver/issues/16
- 
##### prs (is:pr created:>2023-09-28)
- https://github.com/hyperledger-labs/did-webs-resolver/pull/8
- https://github.com/hyperledger-labs/did-webs-resolver/pull/9
- https://github.com/hyperledger-labs/did-webs-resolver/pull/10
- https://github.com/hyperledger-labs/did-webs-resolver/pull/11
- https://github.com/hyperledger-labs/did-webs-resolver/pull/17
- https://github.com/hyperledger-labs/did-webs-resolver/pull/18

**Comment by kentbull** [comment]
SignifyTS credential issuance multisig needs Creder._exhale, which really just calls `sizeify`. Is anyone else working on this?
